packages/create-next-app/templates/index.ts

```diff
@@ -282,6 +282,13 @@ export const installTemplate = async ({
   const devDeps = Object.keys(packageJson.devDependencies).length;
   if (!devDeps) delete packageJson.devDependencies;
 
+  /* For pnpm v10+ x-ref: https://github.com/vercel/next.js/issues/83158 */
+  if (packageManager === "pnpm") {
+    packageJson.pnpm = {
+      onlyBuiltDependencies: ["sharp"],
```

Comment:
> There's still two other postinstall that are ignored if you use tw and eslint's next config:

what about also including the `@tailwindcss/oxide` and `unrs-resolver` here?

since they are trusted and common packages, I would suggest adding them always (unconditionally), but I can understand the approach of adding them conditionally

```suggestion
      onlyBuiltDependencies: [
        "@tailwindcss/oxide",
        "sharp",
        "unrs-resolver",
      ],
```

Reply:
Let's do that in a separate PR since that requires more research into what these other scripts do.

I can approve the `sharp` change now.

Reply:
Ok, here's some starting points for the research:

1. `@tailwindcss/oxide` https://github.com/tailwindlabs/tailwindcss/pull/17929
   1. But it looks like @philipp-spiess just reverted it, because of pnpm v10 requiring manual approval of postinstall scripts https://github.com/tailwindlabs/tailwindcss/pull/19149
2. `unrs-resolver` https://github.com/unrs/unrs-resolver/pull/66
   1. https://github.com/un-ts/napi-postinstall
   2. seems related to a npm <v11 bug only, so ok to ignore
      1. https://github.com/unrs/unrs-resolver/issues/85#issuecomment-2820885808
      2. https://github.com/unrs/unrs-resolver/issues/56#issuecomment-2814258854

Reply:
Added `unrs-resolver`. `@tailwindcss/oxide` is not required anymore due to https://github.com/tailwindlabs/tailwindcss/pull/19149